docs/ensuring-repro/workflows/run-module-gha.md

```diff
@@ -33,10 +38,11 @@ Each module testing GHA is initially created with these steps, which should be u
 
 - Checkout the repository
 - Download test data
-- Setup the module environment
-    - Depending on [the flags used when creating your module](../../contributing-to-analyses/analysis-modules/creating-a-module.md#module-creation-script-flags), this will steps steps needed to install the [`renv` and/or conda environment](../managing-software/index.md) from existing environment files (`renv.lock` and/or `conda-lock.yml`, respectively)
+    - After this step the `data/current` directory will point to the test data, which ensures the module GHA to runs using the test data.
```

Comment:
We probably want to say something about downloading the correct values here?
I used stub links just because I can't find the correct links easily in this interface...
```suggestion
    - Use the [`download-data.py`](#STUB_LINK) and/or [`download-results.py`](#STUB_LINK) scripts to specify the set of input files you need, with the `--test-data` flag to specify downloading the test data.
    - After this step the `data/current` directory will point to the test data, which ensures the module GHA to runs using the test data.
```